satpy/readers/_geos_area.py

```diff
@@ -50,17 +50,25 @@ def make_ext(ll_x, ur_x, ll_y, ur_y, h):
         ll_y: The lower left y coordinate (m)
         ur_y: The upper right y coordinate (m)
         h: The satellite altitude above the Earth's surface
+        round: Round values by 7 decimal places to avoid inconsistencies
+            with floating point calculations.
```

Comment:
Just a suggestion, to improve clearness
```suggestion
        round: Round intermediate area extent angle values by 7 decimal places 
        to avoid inconsistencies with floating point calculations.
```
(the same to the `get_area_extent` docstring). 

I also notice now that the docstring here is wrong, since the input values are in `deg` instead of the documented `m`.